Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7-slim
+LABEL "project.home"="https://github.com/BitBotFactory/poloniexlendingbot"
+
+#
+# Build: docker build -t <your_id>/pololendingbot .
+# Run: docker run -d -v /pololendingbot_data:/data -p 8000:8000 <your_id>/pololendingbot
+#
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r ./requirements.txt
```

Comment:
Should this be CMD so the requirements get updated? Maybe not, I'm not sure. I do think all the RUNs should be on one line though to reduce the number of layers in the image.